lib/screens/form/sections/opinion_field.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_form_exercise/screens/form/sections/build_layout.dart';
+
+import '../../app_paddings.dart';
+import '../form_page_utils.dart';
+
+class OpinionField extends StatelessWidget {
+  const OpinionField({
+    Key? key,
+    required ValueSetter<String?> onChanged
+  }) : _onChanged = onChanged, super(key: key);
+  final ValueSetter<String?> _onChanged;
+  
+  static const double _opinionTextBoxHeight = 100;
+  static const int _opinionTextBoxMaxLines = 5;
```

Comment:
These can be at the bottom and not a part of the class. Further, you don't need the `opinionTextBox` prefix since it's only used here.